docs/internals/security.txt

```diff
@@ -91,15 +117,9 @@ email message, signed with the Django release key, containing:
 * The date on which the Django team will apply these patches, issue
   new releases and publicly disclose the issue.
 
-Simultaneously, the reporter of the issue will receive notification of
```

Comment:
Again, just updating based on current practices. I haven't done this as long as I've been doing releases. I'm not sure why reporters would need special notification.